code/game/objects/structures/tables_racks.dm

```diff
@@ -294,6 +299,59 @@
 	..()
 	return SECONDARY_ATTACK_CONTINUE_CHAIN
 
+/obj/structure/table/attack_hand_secondary(mob/user, list/modifiers)
+	. = ..()
+	if(!istype(user) || !user.can_interact_with() || isobserver(user))
+		return
+	if(!can_flip)
+		return
+	user.balloon_alert_to_viewers("flipping table...")
+	if(!do_after(user, max_integrity * 0.25))
+		return
+
+	var/obj/structure/flippedtable/flipped_table = new flipped_table_type(loc)
+	flipped_table.name = "flipped [name]"
+	flipped_table.desc = "[desc]<br> It's been flipped on its side!"
+	flipped_table.icon_state = base_icon_state
+	var/new_dir = get_dir(user, flipped_table)
+	flipped_table.dir = new_dir
+	if(new_dir == NORTH)
+		flipped_table.layer = BELOW_MOB_LAYER
+	flipped_table.max_integrity = max_integrity
+	flipped_table.update_integrity(.get_integrity())
```

Comment:
bro are you not even compiling this

Reply:
Nope! I this _was_ going to be my last commit for the day. in restrospect. should have debugged first before trying to hit the sack